### DISCUSSION TITLE
Upgrade Versions Help

### DISCUSSION THREAD

**Discussion by vzamora** [userdocs/qbittorrent-nox-static#144] [Q&A] [answered]
How do I upgrade versions without losing what I'm seeding?

Running `curl -sL https://github.com/userdocs/qbittorrent-nox-static/releases/latest/download/dependency-version.json`

I get:

    "openssl": "3.3.0",

    "boost": "1.85.0",

    "libtorrent_1_2": "1.2.19",

    "libtorrent_2_0": "2.0.10",

    "qt5": "5.15.13",

    "qt6": "6.7.0",

    "qbittorrent": "4.6.4",

    "revision": "4"

The weird thing is it lists "qbittorrent":"4.6.4" but if I go to the qBit WebUI and go to Help > About, it lists me as being on 4.5.2.  A Linux ISO site also shows my client as being 4.5.2.

![image](https://github.com/userdocs/qbittorrent-nox-static/assets/449075/aeace6d7-5b56-41a0-ad3f-9d73a0909141)

**Comment by userdocs** [comment, marked as answer]
stop qbittorrent, replace binary and start it again.

Nothing should be lost.
